app/design/frontend/Magento/blank/Magento_Catalog/web/css/source/_module.less

```diff
@@ -813,6 +813,7 @@
             &.delete {
                 &:extend(.abs-remove-button-for-blocks all);
                 left: -6px;
+                right: auto;
                 position: absolute;
                 top: 0;
             }
```

Comment:
still one line around 457 have style on delete button. You should clean that to avoid confusing in future
You can indicate that line with property right: 0; and position: absolute

Reply:
Also please sort property alphabetically

Reply:
Done.
Weird that here (in this preview) the latest changes aren't reflected.